pageserver/src/metrics.rs

```diff
@@ -10,20 +10,14 @@ use pageserver_api::models::state;
 use utils::id::{TenantId, TimelineId};
 
 /// Prometheus histogram buckets (in seconds) that capture the majority of
-/// latencies in the microsecond range but also extend far enough up to distinguish
-/// "bad" from "really bad".
+/// latencies in the microsecond and millisecond range but also extend far
+/// enough up to distinguish "bad" from "really bad".
 fn get_buckets_for_critical_operations() -> Vec<f64> {
-    let buckets_per_digit = 5;
-    let min_exponent = -6;
-    let max_exponent = 2;
-
-    let mut buckets = vec![];
-    // Compute 10^(exp / buckets_per_digit) instead of 10^(1/buckets_per_digit)^exp
-    // because it's more numerically stable and doesn't result in numbers like 9.999999
-    for exp in (min_exponent * buckets_per_digit)..=(max_exponent * buckets_per_digit) {
-        buckets.push(10_f64.powf(exp as f64 / buckets_per_digit as f64))
-    }
-    buckets
+    vec![
+        0.000_001, 0.000_010, 0.000_100, // 1 us, 10 us, 100 us
+        0.001_000, 0.010_000, 0.100_000, // 1 ms, 10 ms, 100 ms
+        1.0, 10.0, 100.0, // 1 s, 10 s, 100 s
+    ]
```

Comment:
It was a bit hard to see in the old code, but this used to be 42 buckets, as follows:
```
0.000001
0.000001584893192461114
0.0000025118864315095823
0.000003981071705534969
0.00000630957344480193
0.00001
0.00001584893192461114
0.000025118864315095822
0.000039810717055349695
0.00006309573444801929
0.0001
0.00015848931924611142
0.00025118864315095795
0.00039810717055349735
0.000630957344480193
0.001
0.001584893192461114
0.0025118864315095794
0.003981071705534973
0.00630957344480193
0.01
0.015848931924611134
0.025118864315095794
0.039810717055349734
0.06309573444801933
0.1
0.15848931924611134
0.251188643150958
0.3981071705534972
0.6309573444801932
1
1.5848931924611136
2.51188643150958
3.9810717055349722
6.309573444801933
10
15.848931924611133
25.118864315095795
39.810717055349734
63.09573444801933
100
+Inf
```
(The `+Inf` bucket doesn't need to be listed here, it is created implicitly)